firefox-ios/Client/Assets/RemoteSettingsData/RemoteSettingsFetchConfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "rules": [
+    {
+      "name": "Password Rules",
+      "url": "https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/password-rules/records",
+      "file": "./firefox-ios/Client/Assets/RemoteSettingsData/RemotePasswordRules.json"
+    }
```

Comment:
Making it ready for review, once this gets merged I will start the process of adding more files for upcoming remote settings work. So think of this as the initial step